optuna/study/_frozen.py

```diff
@@ -0,0 +1,94 @@
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Sequence
+
+from optuna import logging
+from optuna.study._study_direction import StudyDirection
+
+
+_logger = logging.get_logger(__name__)
+
+
+class FrozenStudy:
+    """Basic attributes of a :class:`~optuna.study.Study`.
```

Comment:
Do you think if it is possible to clarify what kind of attributes of `Study` are basic?
For example, `FrozenTrial` shows that it contains status and results of a `Trial`.
https://optuna.readthedocs.io/en/stable/reference/generated/optuna.trial.FrozenTrial.html

Reply:
This comment comes from `StudySummary`, and we'll keep this comment as it.
https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.StudySummary.html#optuna.study.StudySummary